netbsd: no readline

Python build missing readline, likely:
```
    def import_paperkey(self, args):
        # imported here because it has global side effects
>       import readline
E       ImportError: No module named 'readline'

.tox/py34/lib/python3.4/site-packages/borg/crypto/keymanager.py:146: ImportError
```

--- a/src/borg/crypto/keymanager.py
+++ b/src/borg/crypto/keymanager.py
@@ -142,8 +142,11 @@ class KeyManager:
         self.store_keyblob(args)
 
     def import_paperkey(self, args):
-        # imported here because it has global side effects
-        import readline
+        try:
+            # imported here because it has global side effects
+            import readline
+        except ImportError:
+            print('Note: No line editing available due to missing readline support')
 
         repoid = bin_to_hex(self.repository.id)[:18]
         try:
@@ -151,8 +154,8 @@ class KeyManager:
                 # id line input
                 while True:
                     idline = input('id: ').replace(' ', '')
-                    if idline == "":
-                        if yes("Abort import? [yN]:"):
+                    if idline == '':
+                        if yes('Abort import? [yN]:'):
                             raise EOFError()
 
                     try:
@@ -184,8 +187,8 @@ class KeyManager:
                 while True:
                     inline = input('{0:2d}: '.format(idx))
                     inline = inline.replace(' ', '')
-                    if inline == "":
-                        if yes("Abort import? [yN]:"):
+                    if inline == '':
+                        if yes('Abort import? [yN]:'):
                             raise EOFError()
                     try:
                         (data, checksum) = inline.split('-')

